src/patternfly/components/Popover/popover.scss

```diff
@@ -47,30 +50,31 @@
   --#{$popover}__arrow--m-left--Rotate: 45deg;
 
   // Close
-  --#{$popover}__close--Top: calc(var(--#{$popover}__content--PaddingTop) - var(--#{$pf-global}--spacer--form-element)); // align top of button with top of text
-  --#{$popover}__close--Right: calc(var(--#{$popover}__content--PaddingRight) - var(--#{$pf-global}--spacer--md)); // align right of content
-  --#{$popover}__close--sibling--PaddingRight: var(--#{$pf-global}--spacer--2xl);
+  --#{$popover}__close--Top: calc(var(--#{$popover}__content--PaddingTop) - var(--pf-t--global--spacer--sm)); // align top of button with top of text
+  --#{$popover}__close--Right: calc(var(--#{$popover}__content--PaddingRight)); // align right of content
+  --#{$popover}__close--sibling--PaddingRight: var(--pf-t--global--spacer--2xl);
 
   // Header
-  --#{$popover}__header--MarginBottom: var(--#{$pf-global}--spacer--sm);
+  --#{$popover}__header--MarginBottom: var(--pf-t--global--spacer--sm);
 
   // Title text
-  --#{$popover}__title-text--LineHeight: var(--#{$pf-global}--LineHeight--md);
-  --#{$popover}__title-text--FontFamily: var(--#{$pf-global}--FontFamily--heading);
-  --#{$popover}__title-text--FontSize: var(--#{$pf-global}--FontSize--md);
+  --#{$popover}__title-text--LineHeight: var(--pf-t--global--font--line-height--heading);
```

Comment:
Could use another pair of eyes, but it looks like the alignment of the icon and title text is a little off with this line height, whereas if I used the `...--body` line height it looks more in line. Which line height should be used here?

Reply:
Update the title text to body--lg and remove the title text line height